pkg/cmd/outputs.go

```diff
@@ -47,7 +47,7 @@ func NewSetOutputCommand() *cobra.Command {
 				return err
 			}
 
-			fmt.Fprintln(cmd.OutOrStdout(), "Successfully set output.")
+			fmt.Fprintln(cmd.OutOrStdout(), `Successfully set output for "`+key+`".`)
```

Comment:
I would use a format string here. I find concatenation to be a little clunky.

```suggestion
			if _, err := fmt.Fprintf(cmd.OutOrStdout(), "Successfully set output for %s.\n", key); err != nil {
			    return err
			}
```

 https://golang.org/pkg/fmt/#Fprintf